src/components/OrgSettings/General/OrgUpdate/OrgUpdate.tsx

```
@@ -238,108 +250,48 @@
+            {tCommon('displayImage')}:
+          </Form.Label>
+          <Form.Control
+            className={styles.customFileInput}
+            accept="image/*"
+            placeholder={tCommon('displayImage')}
+            name="photo"
+            type="file"
+            multiple={false}
+            onChange={async (e: React.ChangeEvent): Promise<void> => {
+              const target = e.target as HTMLInputElement;
+              const file = target.files && target.files[0];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)